config/software/cpanminus.rb

```diff
@@ -13,6 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+# do_not_auto_update
```

Comment:
I don't see a reason why we'd need this. I think we're safe to update the default version to 1.9019.